engine/src/main/junit-shared/org/operaton/bpm/engine/test/util/JobExecutorWaitUtils.java

```diff
@@ -310,7 +310,7 @@ private static long numberOfJobsAvailable(ManagementService managementService) {
    *
    * @return the default {@link ProcessEngine} instance as configured in the BPM platform.
    */
-  private static ProcessEngine getProcessEngine() {
+  public static ProcessEngine getProcessEngine() {
```

Comment:
```suggestion
 static ProcessEngine getProcessEngine() {
```
Package protected should be enough